mettle/worker/worker.go

```diff
@@ -122,19 +122,16 @@ func RunOne(instanceName, name, storage, dir, cacheDir, sandbox, altSandbox, tok
 	if err != nil {
 		return err
 	}
-	// Have to do this async since mempubsub doesn't seem to store messages?
-	go func() {
-		time.Sleep(50 * time.Millisecond) // this is dodgy obvs
-		b, _ := proto.Marshal(&pb.ExecuteRequest{
-			InstanceName: instanceName,
-			ActionDigest: digest,
-		})
-		log.Notice("Sending request to build %s...", digest.Hash)
-		if err := topic.Send(context.Background(), &pubsub.Message{Body: b}); err != nil {
-			log.Fatalf("Failed to submit job to internal queue: %s", err)
-		}
-		log.Notice("Sent request to build %s", digest.Hash)
```

Comment:
It might be worth keeping this log line and the one above.

Reply:
Yeah fair, I think it is less of a deal now it's synchronous but doesn't hurt to keep them.